client/reader/lib/author-name-blacklist/index.ts

```diff
@@ -12,7 +11,7 @@ const authorNameBlacklist = [ 'admin' ];
  * @param {string} authorName Author name
  * @returns {boolean} True if blacklisted
  */
-export const isAuthorNameBlacklisted = authorName => {
+export const isAuthorNameBlacklisted = ( authorName: string ): boolean => {
 	if ( ! authorName ) {
 		return false;
 	}
```

Comment:
although it's not the intention of this PR to preform a general refactor, we should consider inlining the blacklist check to avoid needless memory allocation and array operations

```js
return 'admin' === authorName.toLowerCase();
```